client/my-sites/activity/activity-log-v2/index.tsx

```diff
@@ -47,10 +50,14 @@ const ActivityLogV2: FunctionComponent = () => {
 		( state ) => siteId && siteHasFeature( state, siteId, WPCOM_FEATURES_FULL_ACTIVITY_LOG )
 	);
 	const settingsUrl = useSelector( ( state ) => getSettingsUrl( state, siteId, 'general' ) );
+	let upgradePlanNames = translate( 'Jetpack Personal or Jetpack Premium' );
 
 	let upsellURL;
 	if ( hasJetpackPartnerAccess && ! isA8CForAgencies() ) {
 		upsellURL = `/partner-portal/issue-license?site_id=${ siteId }`;
+	} else if ( ! isA8CForAgencies() && isSimple ) {
+		upsellURL = `https://wordpress.com/checkout/${ siteSlug }/${ PLAN_PERSONAL }`;
+		upgradePlanNames = 'Starter plan';
```

Comment:
Could we get this plan name with the dynamic plan name value instead of hardcoding?